package.json

```diff
@@ -169,7 +169,7 @@
     "postinstall": "pwsh -nop -f ./scripts/mp-download.ps1 -copyonly",
     "download-native": "pwsh -nop -f ./scripts/mp-download.ps1 -ElectronVersions 11.2.1 -IgnoreNodeVersion",
     "build": "babel src --out-dir lib --source-maps",
-    "rebuild": "electron-rebuild",
+    "install": "electron-rebuild -v 13.1.7",
```

Comment:
@jakobrosenberg I think it better to keep it `rebuild` to not force the end-user to use electron-rebuild on his end